src/css/home.scss

```diff
@@ -248,7 +252,7 @@
       }
 
       .greenCircle {
-        right: 0;
+        left: 226px;
```

Comment:
@astanovferuz , it should be `right: 0` for the responsiveness. Or, we can make it with `%` instead of fixed offset like as `left: 70%`.